src/dotnet-archive/dotnet-archive.csproj

```diff
@@ -6,7 +6,7 @@
     <TargetFramework>netcoreapp2.0</TargetFramework>
     <OutputType>Exe</OutputType>
     <EnableApiCheck>false</EnableApiCheck>
-    <RuntimeIdentifier>win7-x64</RuntimeIdentifier>
```

Comment:
For this package it's fine. This is only temporary (and it isn't built by our CI, I just manually build it). @livarcocc is creating a separate repo for the archiver so we can share the code properly.